packages/react-core/src/next/components/Wizard/WizardStep.tsx

```diff
@@ -30,7 +30,7 @@ export interface WizardStepProps {
   footer?: React.ReactElement | Partial<WizardFooterProps>;
   /** Used to determine icon next to the step's navigation item */
   status?: 'default' | 'error';
-  /** Flag to determine whether sub-steps can collapse or not */
+  /** Flag to determine whether sub-steps can collapse or not. Defaults to true. */
   isCollapsible?: boolean;
```

Comment:
I don't feel strongly about what the default should be. My only concern is how this will effect existing consumers of the wizard if they transition to the Next component.

Reply:
Switching to the Next component will require Consumers to reimplement their wizards.  This prop is one of many API changes the consumer will need to make. So changing the name an default is not a big deal in the grand scheme of things.  
I do think `isExpandable` is a more consistent name that we use for similar interactions in multiple components.   So I would suggest that for name.  
As for the default, my question for design is do we have s preference as to whether sub-steps should se expandable by default should that be opt in?  (It was opt in for original wizard).  If there is no preference, we can leave it as is in this PR.

Reply:
I went ahead and made the change to have it default to "false" for now. I don't have a preference and I can switch it back if we want.